cores/esp8266/Schedule.cpp

```diff
@@ -6,25 +6,31 @@
 #include "interrupts.h"
 #include "coredecls.h"
 
-typedef std::function<bool(void)> mFuncT;
-
+typedef std::function<void(void)> mSchedFuncT;
 struct scheduled_fn_t
 {
     scheduled_fn_t* mNext = nullptr;
-    mFuncT mFunc;
-    esp8266::polledTimeout::periodicFastUs callNow;
-    schedule_e policy;
-
-    scheduled_fn_t() : callNow(esp8266::polledTimeout::periodicFastUs::alwaysExpired) { }
+    mSchedFuncT mFunc;
 };
 
 static scheduled_fn_t* sFirst = nullptr;
 static scheduled_fn_t* sLast = nullptr;
 static scheduled_fn_t* sUnused = nullptr;
 static int sCount = 0;
 
+typedef std::function<bool(void)> mRecFuncT;
+struct recurrent_fn_t
+{
+    recurrent_fn_t* mNext = nullptr;
+    mRecFuncT mFunc;
+    esp8266::polledTimeout::periodicFastUs callNow;
+    recurrent_fn_t (esp8266::polledTimeout::periodicFastUs interval): callNow(interval) { }
+};
+
+static recurrent_fn_t* rFirst = nullptr; // fifo not needed
+
 IRAM_ATTR // called from ISR
-static scheduled_fn_t* get_fn_unsafe()
+static scheduled_fn_t* get_fn_unsafe ()
 {
     scheduled_fn_t* result = nullptr;
     // try to get an item from unused items list
```

Comment:
Very minor, but maybe the comment should be moved above the function?  get_fn_unsafe is a bit non-obvious to me.  Something like

````
// Returns a pointer to an unused sched_fn_t, or if none are available allocates a new one.
````